include/ettus/rfnoc_block_impl.h

```diff
@@ -74,7 +74,8 @@ namespace gr {
               const device3::sptr &dev,
               const std::string &block_id,
               const ::uhd::stream_args_t &tx_stream_args,
-              const ::uhd::stream_args_t &rx_stream_args
+              const ::uhd::stream_args_t &rx_stream_args,
+              const bool end_of_burst_enabled = true
```

Comment:
I think this name is misleading. How about `enable_eob_on_stop`?

Reason being, this made take another look at this code, and we should be honoring EOBs in the TX functions, but we don't. Let's assume that we did, then this argument would imply that all EOBs are disabled, but we only want to disable EOBs on stop().

Reply:
Hey @mbr0wn ,
That makes sense, I honestly did not think of things from that perspective. I have updated the name accordingly.